src/@types/ui-lib.d.ts

```diff
@@ -0,0 +1,124 @@
+import {
```

Comment:
Do we have to add this file in the portal? Is it possible to install @type dependency (like `@types/astar-ui`) under devDependencies?